src/app/common/common.tsx

```diff
@@ -1070,6 +1071,7 @@ class Dropdown extends React.Component<DropdownProps, DropdownState> {
             <div
                 className={cn(`wave-dropdown ${className || ""}`, {
```

Comment:
change this to cn("wave-dropdown", className), ...)
(no need for the `|| ""`) in that case